whitepapers/business-value/glossary.md

```diff
@@ -1 +1,17 @@
-This glossary will serve as a reference guide to assist non-technical managers to map key terms, concepts, and acronyms related to open source in business environments.
+# The Glossary for you!
+This file will serve as a reference guide to assist non-technical managers to understand key terms, concepts, and acronyms related to open source in business environments.
+
+# The Acronyms
+TODO - Talk Openly, Develop Openly.
```

Comment:
As this is not related to the content of the oss in business guide, I'd remove 🙂